extensions/EXT_float_blend/extension.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<draft href="EXT_float_blend/">
+<extension href="EXT_float_blend/">
```

Comment:
Shouldn't there be a public_webgl post about this first?

Reply:
I can post about it, but I don't think there's an ordering requirement.